unit_test/tstBatchedLinearAlgebra.hpp

```diff
@@ -1021,6 +1021,15 @@ void vectorTest()
     EXPECT_EQ( x.stride( 0 ), 1 );
     EXPECT_EQ( x.extent( 0 ), 3 );
 
+    // normalize
+    auto x_copy = x;
```

Comment:
Why make the copy here?

Reply:
does not want to change the original `x` because it is used in the followings..

Reply:
But this is a shallow copy, isn't it?

Reply:
No, I'm wrong